jest.cjs.config.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     '!packages/*/__fixtures__/**',
     '!packages/*/test-utils.js',
     '!packages/*/test-results/**',
+    '!**/__fixtures__/**',
```

Comment:
Is this not duplicative of line 23? If that one doesn't solve the needs for this, then it should be removed as this would cover it.